statchatexpress/database.js

```diff
@@ -0,0 +1,6 @@
+import { Sequelize } from 'sequelize';
+
+export const sequelize = new Sequelize('statchat', 'dswcodes', 'Samuel420', {
```

Comment:
This is the username and password! We don't want to commit this to the repo since anyone can see it. 

To solve this we want to have this info in a different file that's only available locally. One way to do this is to install the dotenv package (check the link here for usage): https://www.npmjs.com/package/dotenv 

```suggestion
import 'dotenv/config';

export const sequelize = new Sequelize('statchat', process.env.DB_USERNAME, process.env.DB_PASSWORD, {
```